tests/console/sshd.pm

```diff
@@ -79,7 +79,7 @@ sub run {
     assert_script_run("echo \"PS1='# '\" >> ~$ssh_testman/.bashrc") unless check_var('VIRTIO_CONSOLE', '0');
 
     # Make interactive SSH connection as the new user
-    type_string "expect -c 'spawn ssh $ssh_testman\@localhost -t;expect \"Are you sure\";send yes\\n;expect sword:;send $ssh_testman_passwd\\n;interact'\n";
+    type_string "expect -c 'spawn ssh $ssh_testman\@localhost -t;expect \"Are you sure\";send yes\\n;expect sword:;send $ssh_testman_passwd\\n;expect #:;send \\n;interact'\n";
```

Comment:
Why `expect #:` looks like it works, I'm just curious.

Reply:
With `#:` you are adding ~10 sec sleep because it will not match and expect will just timeout and continue ;)

```
desktop:/media/data3/git/metadata # time expect -c 'spawn ssh localhost;expect sword:;send $password\n;expect #;send whoami\n;expect root;expect #;send exit\n;interact'
spawn ssh localhost
Password: 
Last login: Fri Dec 11 12:14:30 2020 from ::1
Have a lot of fun...
desktop:~ # whoami
root
desktop:~ # exit
logout
Connection to localhost closed.

real	0m0.302s
user	0m0.007s
sys	0m0.001s


desktop:/media/data3/git/metadata # time expect -c 'spawn ssh localhost;expect sword:;send $password\n;expect #:;send whoami\n;expect root;expect #:;send exit\n;interact'
spawn ssh localhost
Password: 
Last login: Fri Dec 11 12:14:46 2020 from ::1
Have a lot of fun...
desktop:~ # whoami
root
desktop:~ # exit
logout
Connection to localhost closed.

real	0m20.310s
user	0m0.006s
sys	0m0.004s
```

Reply:
Thanks.  I have not notice my mistake make it seems work:)
I have updated to correct "expect #" and restart case, let wait the result(But this issue seems not reproduce 100%):
http://openqa.suse.de/t5161430

Reply:
let's have a try in osd env.